src/shared/route-urls.ts

```diff
@@ -50,5 +50,7 @@ export enum RouteUrls {
   // Send crypto asset routes
   SendCryptoAsset = '/send',
   SendCryptoAssetForm = '/send/:symbol',
-  SendCryptoAssetFormConfirmation = '/send/:symbol/confirmation',
+  SendBtcCryptoCurrencyConfirmation = '/send/btc/confirmation',
+  SendStxCryptoCurrencyConfirmation = '/send/stx/confirmation',
+  SendStacksSip10Confirmation = '/send/:symbol/confirmation',
```

Comment:
Just wondering if it's not possible to use a single `confirmation` route here? Where it'll implicitly go to the correct place when routing to `confirmation` relative to the already active `/currency` route?

Not necessary, just wondering if possible

Reply:
Would def be better, but let me check if possible ...not sure.